docs/content/en/docs/concepts/tracing-policy/hooks.md

```diff
@@ -400,6 +400,61 @@ The `maxData` flag does not work with `returnCopy` flag at the moment, so it's
 usable only for syscalls/functions that do not require return probe to read the
 data.
 
+### Advanced usage
```

Comment:
I would maybe be more specific on this one, like "extracting structure fields/attribute", whatever you prefer that describes best this feature.